doc/source/handling-files/overlaps.rst

```diff
@@ -144,6 +144,29 @@ command contain an ``rm`` statement, as such:
 This would cause later builds of ``runtime-only.bst`` to no longer
 conflict on the given file.
 
+Changing the location
```

Comment:
This section doesn't fit with the rest of the document. The document discusses an example project, while this section discusses a theoretical example that isn't well defined.

I'm not sure what's the best way to deal with this. Extend the example project so that this can be applied? That could be too complicated for this tutorial. Move it to somewhere else in the documentation? I don't know where that could be.